paragraph.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@editorjs/paragraph';
```

Comment:
This file should be generated during build time and should contain all the types. Investigate on ways to do this, maybe use [this vite plugin ](https://github.com/qmhc/vite-plugin-dts)